coalib/processes/BearRunning.py

```diff
@@ -5,8 +5,8 @@
 from coalib.bears.GlobalBear import GlobalBear
 from coalib.bears.LocalBear import LocalBear
 from coalib.misc import Constants
-from coalib.processes.communication.LogMessage import LOG_LEVEL, LogMessage
 from coalib.processes.CONTROL_ELEMENT import CONTROL_ELEMENT
+from coalib.processes.communication.LogMessage import LOG_LEVEL, LogMessage
```

Comment:
Ugh, I'll have to be more careful about this. Actually we should have a bear taking care of this. What do you think ?